spec/doubleLinkedList_test.js

```diff
@@ -0,0 +1,156 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import DoubleLinkedList from '../src/doubleLinkedList'
+
+chai.use(chaiChange)
+
+describe('Double Linked List', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(DoubleLinkedList).to.be.a('function')
+  })
+
+  describe('insert()', () => {
+    it('Inserts a node (with the provided data) to the tail of the list', () => {
+      const doubleLinkedList = new DoubleLinkedList()
+
+      expect(() => doubleLinkedList.insert('foo'))
+        .to.alter(() => doubleLinkedList.count, { from: 0, to: 1 })
+      expect(doubleLinkedList.getHeadNode().data)
+        .to.equal('foo')
+      expect(() => doubleLinkedList.insert('boo'))
+        .to.alter(() => doubleLinkedList.count, { from: 1, to: 2 })
+       expect(doubleLinkedList.getTailNode().data)
+        .to.equal('boo')
+    })
+  })
+
+  describe('insertFirst()', () => {
+    it('Inserts a node (with the provided data) to the head of the list', () => {
+      const doubleLinkedList = new DoubleLinkedList()
+
+      expect(() => doubleLinkedList.insertFirst('foo'))
+        .to.alter(() => doubleLinkedList.count, { from: 0, to: 1 })
+      expect(() => doubleLinkedList.insertFirst('boo'))
+        .to.alter(() => doubleLinkedList.count, { from: 1, to: 2 })
+        console.log(doubleLinkedList)
+      expect(doubleLinkedList.getHeadNode().data)
+        .to.equal('boo')
+      expect(doubleLinkedList.getTailNode().data)
+        .to.equal('foo')
+    })
+  })
+  describe('contains()', () => {
+    it('Determines whether or not the list contains the provided data', () => {
+      const doubleLinkedList = new DoubleLinkedList()
+      doubleLinkedList.insertFirst('foo')
+      doubleLinkedList.insertFirst('boo')
+
+      expect(doubleLinkedList.contains('foo'))
```

Comment:
I try to create tests that test only one assertion - the `true` and `false` cases are two different tests that are not necessarily related.  If you somehow manage to "break" the true path, this test would count as failed, even though the second assertion may not be failing...  Isolating assertions in individual tests helps you granularly test, and explicitly identify broken functionality with the test suite.